spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
`ENV['RAILS_ENV']` に `development` が入ってる場合はdeveloopment環境で動作してしまうので、下記のようにした方がいいと思います。

```suggestion
ENV['RAILS_ENV'] = 'test'
```